src/spinner.tsx

```diff
@@ -26,6 +26,7 @@ const Spinner: React.FC<Props> = ({color, delay = '500ms', size = 24, style, rol
                 aria-labelledby={spinnerId}
                 className={styles.spinnerIos}
                 height={size}
+                display="block"
```

Comment:
hmmm... why dont set this in css styles?

Reply:
fixed